env/default/default.properties

```diff
@@ -42,13 +42,13 @@ CHANNEL = latest
 CATALOG_SOURCE = redhat-operators
 SUBSCRIPTION_NAME = openshift-pipelines-operator-rh
 
-RESULTS_VERSION = v0.11
-HUB_VERSION = v1.17
+RESULTS_VERSION = v0.11.0
+HUB_VERSION = v1.17.0
 TKN_CLIENT_VERSION = 0.32
-PAC_VERSION = 0.27
-PIPELINE_VERSION = v0.62
-CHAINS_VERSION = v0.21
-TRIGGERS_VERSION = v0.28
+PAC_VERSION = 0.27.2
+PIPELINE_VERSION = v0.61.1
+CHAINS_VERSION = v0.21.1
+TRIGGERS_VERSION = v0.28.0
```

Comment:
Why did you add bugfix version everywhere? It's not there on purpose so that we don't need a new branch for each z-stream release of OpenShift Pipelines